utils/nm-dispatcher.in

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/sh
```

Comment:
Should be `/bin/sh`, not `/usr/bin/sh`

Reply:
/usr/bin/sh is actually the preferred shell, and the rpm build will actually change it if it's /bin/sh...

Reply:
Bleh ... It looks like this is new behavior related to the python2->python3 migration in Fedora/RHEL?  I can't find anything specifically documenting the sh path change or the reasoning behind it, although I see they made several changes to this between RHEL6, RHEL7, and RHEL8.

There are two problems here:
1) This script would be useful on non-RPM systems (at least Debian-based systems), so I would like this script to be usable on both (even if we aren't currently packaging it for Debian - we can add that bit later).  But Debian-based systems don't have /usr/bin/sh, so this script won't work on them as-is.  If `rpmbuild` rewrites the shebang automatically, I'm thinking maybe we should leave it /bin/sh in the repo for use on non-RPM systems, and let `rpmbuild` rewrite it for RPMs.
2) I'm really trying to keep things consistent, and we currently have several scripts in mstpd that use /bin/sh ... So if we're going to change it, I think we should change all of the scripts ... But (circling back to the previous problem) changing all of the scripts breaks other systems.  So, again, I'm thinking let's make this /bin/sh and let `rpmbuild` change it if needed.